src/main/java/org/opentripplanner/model/impl/OtpTransitServiceBuilder.java

```diff
@@ -148,40 +150,26 @@ public TripStopTimes getStopTimesSortedByTrip() {
         return serviceIds;
     }
 
-    public OtpTransitService build() {
-        createNoneExistentIds();
-
-        return new OtpTransitServiceImpl(this);
+    public CalendarServiceData buildCalendarServiceData() {
+        return CalendarServiceDataFactoryImpl.createCalendarServiceData(
+                getAgenciesById().values(),
+                getCalendarDates(),
+                getCalendars()
+        );
     }
 
-    private void createNoneExistentIds() {
+    public OtpTransitService build() {
```

Comment:
OtpTransitService and related identifiers are among those I'd like to rename. They just don't seem like "a service that gives / does transit", but rather a transit data source or store. Maybe TransitDataSource or something?

Reply:
I agree the `OtpTransitService` is missleading, but I don´t think `TransitDataSource` is good either - to me it indicate that the data is fetched from some where. I guess I have been brainwashed by all those Enterprise information systems, but maybe `OTPTransitModel` can work - since it does not provide much business logic, just access to the model.